### PR TITLE
feat: finished sentencizer first version

### PR DESCRIPTION
**Pull request by jakobkruse1**

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-description: This is my dummy executor
+name: sentencizer
+alias: Sentencizer
+description: The sentencizer splits text into sentences.
 author: Jina AI Dev-Team (dev-team@jina.ai)
-url: https://github.com/jina-ai/EXECUTOR_REPO_NAME
+url: https://github.com/jina-ai/executor-text-sentencizer
 avatar: https://avatars.githubusercontent.com/u/60539444?s=200&v=4
-keywords: [encoder, Numpy]
+keywords: [text, sentence, split]
```
Comment (nan-wang) [inline manifest.yml]:
```suggestion
keywords: [text, sentencizer]
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
minor change needed. LGTM

File: setup.py [hunk 2]
```diff
@@ -4,18 +4,18 @@
...
+    description='The text sentencizer splits texts into sentences',
+    url='https://github.com/jina-ai/executor-text-sentencizer',
     classifiers=[
         'Programming Language :: Python :: 3',
         'Operating System :: OS Independent',
     ],
-    py_modules=['jinahub.SUB_PACKAGE_NAME.MODULE_NAME'],
-    package_dir={'jinahub.SUB_PACKAGE_NAME': '.'},
+    py_modules=['jinahub.text.sentencizer'],
+    package_dir={'jinahub.text': '.'},
```
Comment (Yongxuanzhang) [inline setup.py]: This should follw 
jinahub.[indexer|encoder|segmenter|ranker|].module_name
so I think we should use segmenter?

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**